mappings/net/minecraft/world/biome/BiomeArray.mapping

```diff
@@ -0,0 +1,22 @@
+CLASS net/minecraft/class_4548 net/minecraft/world/biome/BiomeArray
```

Comment:
I assume this is 3d array of biomes?

Reply:
yes. This is bound to each chunk. it reflects the 3 biome xyz inputs (not block pos) into a position in the 1d array.